pallets/parachain-staking/src/tests/bond_extra_tests.rs

```diff
@@ -353,6 +404,39 @@ mod proxy_signed_candidate_bond_extra {
                 });
         }
 
+        #[test]
+        fn proxy_proof_is_not_valid() {
```

Comment:
Are we just using a bad nonce to cover all invalid proof cases?

Reply:
I will add a test for each parameter that gets signed.